frontend/src/views/container/compose/create/index.vue

```diff
@@ -135,10 +135,10 @@ import DrawerHeader from '@/components/drawer-header/index.vue';
 import { listComposeTemplate, testCompose, upCompose } from '@/api/modules/container';
 import { loadBaseDir } from '@/api/modules/setting';
 import { MsgError } from '@/utils/message';
-import { javascript } from '@codemirror/lang-javascript';
+import { yaml } from '@codemirror/lang-yaml';
 import { oneDark } from '@codemirror/theme-one-dark';
 
-const extensions = [javascript(), oneDark];
+const extensions = [yaml(), oneDark];
 
 const showLog = ref(false);
 const loading = ref();
```

Comment:
As there are no changes made to the given code snippet between December 18th (2021) and today's date, 2024, it should not show any irregularities. No need for an action in terms of potential issues or optimizations.